src/applications/facility-locator/components/ServiceTypeAhead.jsx

```diff
@@ -73,6 +73,12 @@ class ServiceTypeAhead extends Component {
         onChange={this.handleOnSelect}
         defaultSelectedItem={defaultSelectedItem}
         itemToString={renderService}
+        onInputValueChange={(inputValue, stateAndHelpers) => {
```

Comment:
Important note: we are a couple of versions behind on downshift so this API documentation can be found in their git history [here](https://github.com/downshift-js/downshift/blob/531bd8ea83ae5ecc3c163357ac9be4dc721d2060/README.md#defaultinputvalue)